sponsors/api.py

```diff
@@ -0,0 +1,59 @@
+import csv
+
+from django.conf import settings
+
+from rest_framework import serializers
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from sponsors.enums import LogoPlacementChoices, PublisherChoices
+
+
+class LogoPlacementSerializer(serializers.Serializer):
+    publisher = serializers.CharField()
+    flight = serializers.CharField()
+    sponsor = serializers.CharField()
+    description = serializers.CharField()
+    logo = serializers.URLField()
+    start_date = serializers.DateField()
+    end_date = serializers.DateField()
+    sponsor_url = serializers.URLField()
+
+
+# TODO Currently this endpoint only lists sponsors from pypi sponsors CSV.
+# Once we have all sponsorship data input into pydotorg, we should be
+# able to change this view to fetch data from the database instead.
+class LogoPlacementeAPIList(APIView):
```

Comment:
@ewdurbin I noticed other API endpoints also define authentication and permission with:

```python
authentication_classes = (TokenAuthentication,)
permission_classes = (IsStaffOrReadOnly,)
```

I'm not sure if we need this same config for this endpoint. I mean, the permission class shouldn't be required since this is already a read-only endpoint. But what about the token authentication? Should we generate a token and send to Eric? Let me know what you think about that.